test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetNewTemplatesTests.cs

```diff
@@ -0,0 +1,280 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Xml.Linq;
+using FluentAssertions;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace Dotnet.Integration.Test
+{
+    [Collection("Dotnet Integration Tests")]
+    public class DotnetNewTemplatesTests
+    {
+        private readonly MsbuildIntegrationTestFixture _fixture;
+
+        public DotnetNewTemplatesTests(MsbuildIntegrationTestFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        [PlatformFact(Platform.Linux)]
```

Comment:
Is linux intentional here?

Reply:
Yes, since integration test is slow, I just wanted to run on one platform instead of all platform. 
Why I choose Linux is because it's case sensitive so more likely to fail than Windows.
For `WPF`, `Winform` related tests it only run on Windows platform, so platform limitation is necessary.